shared-bindings/adafruit_pixelbuf/__init__.c

```diff
@@ -29,20 +29,22 @@
 #include "py/runtime.h"
 #include "py/objproperty.h"
 
-#include "shared-bindings/_pixelbuf/__init__.h"
-#include "shared-bindings/_pixelbuf/PixelBuf.h"
+#include "shared-bindings/adafruit_pixelbuf/__init__.h"
+#include "shared-bindings/adafruit_pixelbuf/PixelBuf.h"
 
 
 //| """A fast RGB(W) pixel buffer library for like NeoPixel and DotStar
 //|
-//| The `_pixelbuf` module provides the :py:class:`PixelBuf` class to accelerate
+//| The `adafruit_pixelbuf` module provides the :py:class:`PixelBuf` class to accelerate
 //| RGB(W) strip/matrix manipulation, such as DotStar and Neopixel.
 //|
+//| Also available as `_pixelbuf`. This usage has been deprecated.
```

Comment:
`_pixelbuf` is no longer an auto-generated module name. The easiest thing to do here is probably to just do:
```suggestion
//| Also available as ``_pixelbuf``. This usage has been deprecated.
```